currencyii/src/main/java/nl/tudelft/trustchain/currencyii/ui/bitcoin/BitcoinFragment.kt

```diff
@@ -118,23 +155,62 @@ class BitcoinFragment : BaseFragment(R.layout.fragment_bitcoin),
             return
         }
 
-        var walletManager = WalletManagerAndroid.getInstance()
+        val walletManager = WalletManagerAndroid.getInstance()
 
-        walletBalance.text =
-            "${walletManager.kit.wallet().balance.toFriendlyString()}"
-        if (walletManager.params.id === NetworkParameters.ID_MAINNET) {
-            chosenNetwork.text = "Production Network"
-        } else {
-            chosenNetwork.text = "Test Network"
+        walletBalance.text = walletManager.kit.wallet().balance.toFriendlyString()
+        walletEstimatedBalance.text = walletManager.kit.wallet().getBalance(Wallet.BalanceType.ESTIMATED).toFriendlyString()
+        when {
+            walletManager.params.id === NetworkParameters.ID_MAINNET -> {
+                chosenNetwork.text = "Production Network"
+            }
+            walletManager.params.id === NetworkParameters.ID_REGTEST -> {
+                chosenNetwork.text = "RegTest Network"
+            }
+            walletManager.params.id === NetworkParameters.ID_TESTNET -> {
+                chosenNetwork.text = "TestNet Network"
+            }
+            else -> {
+                chosenNetwork.text = "Unknown Network selected"
+            }
         }
         val seed = walletManager.toSeed()
         walletSeed.text = "${seed.seed}, ${seed.creationTime}"
         yourPublicHex.text = walletManager.networkPublicECKeyHex()
         protocolKey.text = walletManager.protocolAddress().toString()
 
+        if (checkTooMuchBitcoin()) {
+            disableGetBitcoinButton()
```

Comment:
Shouldn't we also have else clause here in which we renable the button? for example when we do transaction and balance is below 10 button would now sill be disabled right?